conf/defaults.ini

```diff
@@ -69,6 +69,10 @@ socket = /tmp/grafana.sock
 # CDN Url
 cdn_url =
 
+# Sets the maximum time in minutes before timing out read of an incoming request and closing idle connections.
+# `0` means there is no timeout for reading the request.
+read_timeout = 0
```

Comment:
Is this used? I think it's not read but if it's planned to be used it should [be documented](https://github.com/grafana/grafana/blob/master/docs/sources/administration/configuration.md) and [added in samples](https://github.com/grafana/grafana/blob/master/conf/sample.ini).